lib/friends/introvert.rb

```diff
@@ -719,7 +725,7 @@ def parse_line!(line, line_num:, state:)
 
       begin
         instance_variable_get("@#{stage.id}") << stage.klass.deserialize(line)
-      rescue => ex # rubocop:disable Style/RescueStandardError
+      rescue StandardError => ex
```

Comment:
I'm not sure why Rubocop is picking this up now?

Reply:
Sorry, a little more detail: Rubocop seems to have deleted the comment _and_ added `      rescue StandardError => ex` sll on its own. I'm a little confused as to why it's doing this now rather than in other PRs, but if you're happy then I'm happy :)